diff -ruNx '*~' libgpg-error-1.10/configure.ac libgpg-error-1.10-adding-pc/configure.ac
--- libgpg-error-1.10/configure.ac	2010-10-26 11:13:48.000000000 +0200
+++ libgpg-error-1.10-adding-pc/configure.ac	2011-12-13 09:07:00.033842285 +0100
@@ -175,7 +175,7 @@
 # Substitution
 AC_CONFIG_FILES([Makefile])
 AC_CONFIG_FILES([po/Makefile.in m4/Makefile])
-AC_CONFIG_FILES([src/Makefile tests/Makefile])
+AC_CONFIG_FILES([src/Makefile tests/Makefile src/libgpg-error.pc])
 AC_CONFIG_FILES([lang/Makefile lang/cl/Makefile lang/cl/gpg-error.asd])
 AC_CONFIG_FILES([src/versioninfo.rc])
 
diff -ruNx '*~' libgpg-error-1.10/src/libgpg-error.pc.in libgpg-error-1.10-adding-pc/src/libgpg-error.pc.in
--- libgpg-error-1.10/src/libgpg-error.pc.in	1970-01-01 01:00:00.000000000 +0100
+++ libgpg-error-1.10-adding-pc/src/libgpg-error.pc.in	2011-12-13 09:07:28.591613936 +0100
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${exec_prefix}/include
+
+Name: libgpg-error
+Description: Library of common error values for all GnuPG components
+Version: @VERSION@
+Libs: -L${libdir} -lgpg-error
+Cflags: -I${includedir}
+
diff -ruNx '*~' libgpg-error-1.10/src/Makefile.am libgpg-error-1.10-adding-pc/src/Makefile.am
--- libgpg-error-1.10/src/Makefile.am	2010-09-16 15:09:17.000000000 +0200
+++ libgpg-error-1.10-adding-pc/src/Makefile.am	2011-12-13 09:08:22.753180812 +0100
@@ -38,12 +38,16 @@
 m4datadir = $(datadir)/aclocal
 m4data_DATA = gpg-error.m4
 
+pcfiles = libgpg-error.pc
+pkgconfig_DATA = $(pcfiles)
+pkgconfigdir = $(libdir)/pkgconfig
+
 EXTRA_DIST = mkstrtable.awk err-sources.h.in err-codes.h.in \
 	mkerrnos.awk errnos.in README \
 	mkerrcodes.awk mkerrcodes1.awk mkerrcodes2.awk mkerrcodes.c \
 	mkheader.awk gpg-error.h.in mkw32errmap.c w32-add.h w32ce-add.h \
 	err-sources.h err-codes.h gpg-error-config.in gpg-error.m4 \
-	gpg-error.def.in versioninfo.rc.in
+	gpg-error.def.in versioninfo.rc.in libgpg-error.pc.in
 
 BUILT_SOURCES = err-sources.h err-codes.h code-to-errno.h code-from-errno.h \
 	err-sources-sym.h err-codes-sym.h errnos-sym.h gpg-error.h \
