diff --git a/configure.ac b/configure.ac
index a099aba..4f6a2cd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -544,7 +544,7 @@ AM_CONDITIONAL([BUILD_TESTS], [test "x$build_tests" != xno])
 #
 AC_CONFIG_FILES([Makefile])
 AC_CONFIG_FILES([doc/Makefile po/Makefile.in m4/Makefile])
-AC_CONFIG_FILES([src/Makefile tests/Makefile])
+AC_CONFIG_FILES([src/Makefile tests/Makefile src/libgpg-error.pc])
 AC_CONFIG_FILES([lang/Makefile lang/cl/Makefile lang/cl/gpg-error.asd])
 AC_CONFIG_FILES([src/versioninfo.rc src/gpg-error.w32-manifest])
 AC_CONFIG_FILES([src/gpg-error-config], [chmod +x src/gpg-error-config])
diff --git a/src/Makefile.am b/src/Makefile.am
index 5623aff..f71cdfa 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -82,6 +82,10 @@ bin_SCRIPTS = gpg-error-config
 m4datadir = $(datadir)/aclocal
 m4data_DATA = gpg-error.m4
 
+pcfiles = libgpg-error.pc
+pkgconfig_DATA = $(pcfiles)
+pkgconfigdir = $(libdir)/pkgconfig
+
 EXTRA_DIST = mkstrtable.awk err-sources.h.in err-codes.h.in \
 	mkerrnos.awk errnos.in README \
 	mkerrcodes.awk mkerrcodes1.awk mkerrcodes2.awk mkerrcodes.c \
@@ -89,6 +93,7 @@ EXTRA_DIST = mkstrtable.awk err-sources.h.in err-codes.h.in \
 	err-sources.h err-codes.h gpg-error-config.in gpg-error.m4 \
 	gpg-error.vers gpg-error.def.in \
         versioninfo.rc.in gpg-error.w32-manifest.in \
+	libgpg-error.pc.in \
 	$(lock_obj_pub)
 
 BUILT_SOURCES = err-sources.h err-codes.h code-to-errno.h code-from-errno.h \
diff --git a/src/libgpg-error.pc.in b/src/libgpg-error.pc.in
new file mode 100644
index 0000000..d0f24c5
--- /dev/null
+++ b/src/libgpg-error.pc.in
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${exec_prefix}/include
+
+Name: libgpg-error
+Description: Library of common error values for all GnuPG components
+Version: @VERSION@
+Libs: -L${libdir} -lgpg-error
+Cflags: -I${includedir}
+
